packages/stems/src/components/Button/Button.module.css

```diff
@@ -137,17 +136,17 @@
   fill: var(--primary-secondary-text);
 }
 
-.primary.includeHoverAnimations:hover:enabled {
```

Comment:
headsup this is needed cause anchor tags dont have this pseudoelement, but can be generalized with attaching an "enabled" classname